spec/integ/matrix-client-event-timeline.spec.ts

```diff
@@ -1178,7 +1241,7 @@ describe("MatrixClient event timelines", function() {
                 await flushHttp(client.getEventTimeline(timelineSet, THREAD_ROOT.event_id!));
 
                 respondToMessagesRequest();
-                const timeline = await flushHttp(client.getLatestTimeline(timelineSet));
```

Comment:
It looks like some non-refresh-timeline tests got infected with `getLatestTimeline` usage in the months while these PR's have sat.

But I don't really want to support `getLatestTimeline` for threads which these tests rely on.

Either we have to maintain the old `getLatestTimeline(...)` alongside `getLatestLiveTimeline(...)` forever or we can refactor these tests away from it.